source/_integrations/light.markdown

```diff
@@ -32,7 +32,7 @@ Most lights do not support all attributes. You can check the integration documen
 | `xy_color` | yes | A list containing two floats representing the xy color you want the light to be. Two comma-separated floats that represent the color in XY. You can find a great chart here: [Hue Color Chart](https://developers.meethue.com/documentation/core-concepts#color_gets_more_complicated).
 | `rgb_color` | yes | A list containing three integers between 0 and 255 representing the RGB color you want the light to be. Three comma-separated integers that represent the color in RGB, within square brackets. Note that the specified RGB value will not change the light brightness, only the color.
 | `white_value` | yes | Integer between 0 and 255 for how bright a dedicated white LED should be.
-| `color_temp` | yes | An integer in mireds representing the color temperature you want the light to be.
+| `color_temp` | yes | An integer between 154 and 500 mireds representing the color temperature you want the light to be.
```

Comment:
This depends on the integration and is not a general limitation of the light entity component.